patches/chromium/feat_add_set_theme_source_to_allow_apps_to.patch

```diff
@@ -61,19 +61,6 @@ index edd042cfe327810c120c4b446c043761c0d21b9f..59aaf71ea1e7931c421a46a7ca8cadb4
    DISALLOW_COPY_AND_ASSIGN(NativeTheme);
  };
  
-diff --git a/ui/native_theme/native_theme_dark_aura.cc b/ui/native_theme/native_theme_dark_aura.cc
-index 87b0abc9c646ace8cafc2b83eb4da6ffd3370237..75c677def845711ef4f3414f0a7df4f88431bf25 100644
---- a/ui/native_theme/native_theme_dark_aura.cc
-+++ b/ui/native_theme/native_theme_dark_aura.cc
-@@ -20,6 +20,8 @@ SkColor NativeThemeDarkAura::GetSystemColor(ColorId color_id,
- }
- 
- bool NativeThemeDarkAura::ShouldUseDarkColors() const {
-+  if (theme_source() == ThemeSource::kForcedLight) return false;
-+  if (theme_source() == ThemeSource::kForcedDark) return true;
```

Comment:
@MarshallOfSound can you confirm if this fix is good?

Reply:
@nornagon `native_theme_dark_aura.cc` was removed completely.  As long as this PR is rebased on the latest native theme tests I have confidence in their ability to catch the things we care about